lib/task_helpers/exports/scan_profiles.rb

```diff
@@ -4,11 +4,14 @@ class ScanProfiles
       def export(options = {})
         export_dir = options[:directory]
 
-        ScanItemSet.all.each do |p|
-          next if p.read_only
-          next if p.members.map { |m| m.slice(:filename) }
```

Comment:
What was the intention for this line originally? Is this something we should fix rather than remove?

Reply:
So @branic was saying it's `next if p.read_only` was to stop exporting non editable profiles aka read only.

Reply:
The `next if p.read_only` line is handled by now checking if `--all` was specified. I don't know what the `p.members.map { |m| m.slice(:filename) }` was doing, but in my tests it always returned true and so never exported any profiles.

```
irb(main):006:0> p.members.map { |m| m.slice(:filename) }
=> [{"filename"=>nil}, {"filename"=>nil}, {"filename"=>nil}, {"filename"=>nil}]
```

Reply:
I was working from someone else code to start with so `p.members.map { |m| m.slice(:filename) }` I assumed was needed. Should have done more research. Ho hum.